index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="./css/styles.css">
+  <title>Microverse-Portfolio</title>
+</head>
+
+<body>
+  <div class="wrapper">
```

Comment:
- [ ] Please add some more padding or margin on the sides of this container. If you see on the Figma template there is more space between the edge and the text content; you have some padding but it's not enough. Please implement this 🙏to match the Figma template as closely as you can.
## Your Design
![image](https://user-images.githubusercontent.com/18730106/144294093-c5ff932e-4d51-45ee-b11d-b6314f741640.png)

## Figma Design 
![image](https://user-images.githubusercontent.com/18730106/144294137-e49dcf89-ade8-4612-aedb-78786d51a3c7.png)